cmd/osbuild-playground/playground.go

```diff
@@ -21,7 +22,7 @@ func RunPlayground(img image.ImageKind, d distro.Distro, arch distro.Arch, repos
 	solver.SetDNFJSONPath(findDnfJsonBin())
 
 	// Set cache size to 3 GiB
```

Comment:
hehe, this comment is funny.

Reply:
Indeed it is, I was inclined to remove it entirely but wanted to avoid mixing too many concerns into a single PR. I at least fixed it - thanks for pointing this out :)